extensions/interactions/MultipleChoiceInput/MultipleChoiceInput.py

```diff
@@ -57,6 +57,13 @@ class MultipleChoiceInput(base.BaseInteraction):
             }
         },
         'default_value': [''],
+    }, {
+        'name': 'showChoicesInShuffledOrder',
+        'description': 'Shuffle answer choices',
+        'schema': {
+            'type': 'bool',
+        },
+        'default_value': True
```

Comment:
Ah I see, the default value is also true. So I think it is ok to ignore the previous comment. Is this desired in all situations though? There may be situations in the datastore where the last option might be "All of the above" or "None of the above" or things like that. Where we wouldn't want to shuffle them up right? What's your opinion on this?

Reply:
Default value is set to be true according to the issue thread https://github.com/oppia/oppia/issues/5254#issuecomment-422955181.
For the case of "All of the above" choice, I think it's not an appropriate behavior to shuffle choices. In my opinion, the default value should be false.

Reply:
Hmmm, This is the default option that will be displayed on selecting the interaction. So here I think it is ok. 

However, for the existing explorations, I think it would be better to set it to False. That is, in the migration function, we set it to False. Let me know your opinion.